### PR TITLE
verify-style: run dep check

### PR DESCRIPTION
**Pull request by squeed**
This will catch if we leave the project in a funny state.

### CODE REVIEW

**Comment by squeed** [comment]
Ah, looks like we need to install dep in our `release` tooling first. huh.

File: hack/verify-style.sh [hunk 1]
```diff
@@ -20,6 +20,10 @@ if [[ ! $(which operator-sdk) ]]; then
   echo "operator-sdk not found on PATH. To install:"
   echo "go get -u github.com/operator-framework/operator-sdk/commands/operator-sdk"
 fi
+if [[ ! $(which dep) ]]; then
```
Comment (danwinship) [inline hack/verify-style.sh]: while you're adding this, I'm pretty sure you can remove the check for operator-sdk just above, can't you?

File: hack/verify-style.sh [hunk 2]
```diff
@@ -41,6 +45,9 @@ fi
 echo "Running go vet..."
 go vet $GOPKGS
 
+echo "Running dep check..."
+dep check
+
 # Temporariy disabled because it breaks openshift-ci
```
Comment (danwinship) [inline hack/verify-style.sh]: we should fix this too while we're dealing with things...

File: hack/verify-style.sh (hunk 1 above)
Comment (squeed) [reply hack/verify-style.sh]: definitely - especially since we don't `exit 1` anyways.

File: hack/verify-style.sh (hunk 2 above)
Comment (squeed) [reply hack/verify-style.sh]: hah, it should just be removed, now that our types are in openshift/api

**Comment by squeed** [comment]
Uh, what!? It works on my machine!?

```
Running dep check... 
# vendor is out of sync:
github.com/gogo/protobuf: hash of vendored tree not equal to digest in Gopkg.lock
```

**Comment by squeed** [comment]
Wow, turns out that the .gitignore we got from Github excluded `tags`, not `/tags`. Fixed...

**Pull request merged**